src/steamship_langchain/file_loaders/sphinx.py

```diff
@@ -0,0 +1,114 @@
+import logging
+from datetime import datetime
+from pathlib import Path
+from typing import Dict, List, Optional, Tuple
+
+from steamship import Block, File, MimeTypes, Steamship, Tag
+from steamship.data import TagKind, TagValueKey
+from steamship.data.tags.tag_constants import ProvenanceTag
+
+from steamship_langchain.file_loaders import BaseFileLoader
+
+
+def generate_url_tags(client: Steamship, url: str, metadata: Optional[Dict[str, str]] = None):
+    tags = [
+        Tag(
+            client=client,
+            kind=TagKind.TIMESTAMP,
+            name="timestamp",
+            value={"timestamp": datetime.now().isoformat()},
+        ),
+        Tag(
+            client=client,
+            kind=TagKind.PROVENANCE,
+            name=ProvenanceTag.URL,
+            value={TagValueKey.STRING_VALUE: url},
+        ),
+    ]
+    if metadata:
+        tags.append(Tag(client=client, kind="metadata", name="metadata", value=metadata))
+    return tags
+
+
+class SphinxSiteLoaderBase(BaseFileLoader):
+    """Load sphinx websites into Steamship.
+
+    Note:
+        Requires install of BeautifulSoup, via:
+        `pip install bs4`
+    """
+
+    tag_name: str
+    "Name of html tag to use for imports"
+
+    tag_attributes: Dict[str, str] = {}
+    "Identifying attributes for the tag"
+
+    scheme: str = "https://"
```

Comment:
```suggestion
    SCHEME: str = "https://"
```

I assume this is a constant

Reply:
Not a constant. Allows for `http://` in rare cases. Added a docstring.